src/components/BookItem.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const BookItem = ({ title }) => (
+  <div>
+    <span>{title}</span>
+    <input type="button" value="delete" />
+  </div>
+);
```

Comment:
According to the project requirements, you are suppose to render both the title and author but I see only the title. Kindly consider passing the author also as props and rendering it in the bookItem here. Kindly check [this link](https://pedantic-beaver-cb923b.netlify.app/#/) to help you do that. :wink:

Reply:
Thank you